dbt/models/reporting/schema.yml

```diff
@@ -40,10 +40,7 @@ models:
           expression: |
             cod >= 0
             AND prd >= 0
-            AND (
-              (prb BETWEEN -1 AND 1 AND assessment_stage != 'pre-mailed')
-              OR (prb BETWEEN -1.5 AND 1.5 AND assessment_stage = 'pre-mailed')
-            )
+            AND (prb BETWEEN -1 AND 1 OR assessment_stage = 'pre-mailed')
```

Comment:
Note that there is a comment explaining this choice on lines 37-39 above, it just doesn't appear in the diff by default.